torchao/quantization/README.md

```diff
@@ -36,6 +36,21 @@ model = torchao.autoquant(torch.compile(model, mode='max-autotune'))
 model(input)
 ```
 
+Sometimes it is desirable to reuse a quantization plan that `autoquant` came up with. `torchao.quantization.AUTOQUANT_CACHE` is a dictionary holding autoquant's benchmark results. We can save it and restore it later, which will cause `autoquant` to choose the same quantization methods.
```

Comment:
Hmm really feels like we should be saving this with some module hook automatically